themes/sensei-course-theme/single.php

```diff
@@ -68,6 +68,7 @@
 
 		<!-- wp:sensei-lms/course-theme-notices /-->
 		<!-- wp:sensei-lms/course-content /-->
+		<!-- wp:sensei-lms/quiz-pagination /-->
```

Comment:
Checking the designs, I think we don't want the actions here. Have you added it here for some reason?

Reply:
From the designs. The only one with quiz feedback is this one. I assume the action buttons were supposed to be at the end of the quiz. Which is where I put them. Could you point me to the correct place where I can see the correct designs, please.

<img width="391" alt="Screen Shot 2022-01-12 at 5 52 34 PM" src="https://user-images.githubusercontent.com/2578542/149153074-5233b402-9451-4670-883c-cf2537bb8975.png">

Reply:
Here you can check the designs: o2Xnqd2zgjaMaJFPmvNOx3-fi-20008411%3A49885

I forgot about the discussion on the slack channel that we want the pagination on the quiz results page too. But it's worth to check how it's there in the case of quiz without pagination. Maybe shouldn't we show the "Reset" link in the quiz results page? Or we could think in a better presentation for that.

<img width="976" alt="Screen Shot 2022-01-12 at 11 02 46" src="https://user-images.githubusercontent.com/876340/149154817-d1c98faf-83b6-4fdb-945b-ff68bdac4215.png">

Reply:
> I forgot about the discussion on the slack channel that we want the pagination on the quiz results page too. But it's worth to check how it's there in the case of quiz without pagination. Maybe shouldn't we show the "Reset" link in the quiz results page? Or we could think in a better presentation for that.

It looks a bit out of place when it's alone, I agree. Probably need a better copy, like "Reset Quiz" or "Restart Quiz". We already have a "Restart Quiz" at the top, so maybe we don't need it there. Do we also remove it when the quiz results are with pagination also? Or we show "Reset" link with pagination, but hide when there is no pagination?...

From the blocks/FSE perspective: This is now a "Quiz Actions" block. It's probably better if it does not have varying behavior depending on the context. It would be confusing for the editor/teacher to put this block at the end of quiz results and see it showing nothing at all on the frontend.

Either way, let me know how you want to proceed. I will make adjustments accordingly.

Reply:
Hum. I'd say that keeping it on the bottom of the results page, similar to the quiz page, would be better (also to keep consistency). I think it would seem less "lost" when it has only the "Reset" link. It would also mean adding the footer to this template too. But we don't want it on the lesson page. 🤔

Any idea on that?

Reply:
I think if you don't think of an easy solution for that, after checking the Miroslav issue, you can merge this PR and you can think about that in a new PR. =)